source/MRMesh/MRPointsComponents.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+#include "MRMeshFwd.h"
+#include "MRUnionFind.h"
+#include "MRExpected.h"
+
+namespace MR
+{
+
+namespace PointCloudComponents
+{
+
+/// \defgroup PointClodComponentsGroup PointClodComponents
+/// \ingroup ComponentsGroup
+/// \{
+
+
+///  returns the union of point cloud components containing more then \param minSize points and connected by a distance no greater than \param maxDist
```

Comment:
then -> than

Reply:
if "min"Size then more or equal (not strictly more)

Reply:
fixed